hermes-api/src/main/java/pl/allegro/tech/hermes/api/AuthenticationType.java

```diff
@@ -0,0 +1,7 @@
+package pl.allegro.tech.hermes.api;
+
+public enum AuthenticationType {
```

Comment:
i don't think it's authentication, but authorization in general (in all class names)